### DISCUSSION TITLE
Please allow us to cmd/ctrl-click results from the search fields

### DISCUSSION THREAD

**Discussion by vesper8** [laravel/nova-issues#4623] [Ideas & Feature Requests] [answered]
Title says it is, currently you cannot open the results from the Nova search fields in a new tab by using cmd/ctrl-click

Even though this is not a straight up href link and uses vue/inertia for routing, it's definitely possible, and easy even, to capture the keypress upon click and if the cmd/ctrl/meta key is pressed down then trigger a window.open. I've done this before on Vue-router apps.

Please add this convenience. Many thanks

**Comment by vesper8** [comment]
I'm really dying for this cmd-click functionality on the global search, so I am overriding the GlobalSearch Vue component and it was incredibly easy to add.

All I had to do was these three tiny changes and you can now open in a new tab while holding cmd/ctrl, and if you don't then it opens in the current tab as normal.

It would be really awesome if you could add this small but terribly useful fix.

before
```
@click="goToSelectedResource(item)"
```


after

```

@click="goToSelectedResource($event, item)"
```


before (inside goToCurrentlySelectedResource method)
```
this.goToSelectedResource(resource)
```


after (inside goToCurrentlySelectedResource method)
```
this.goToSelectedResource(event, resource)
```


before
```
    goToSelectedResource(resource) {
      if (this.canceller !== null) this.canceller()

      this.closeSearch()

      let url = Nova.url(
        `/resources/${resource.resourceName}/${resource.resourceId}`
      )

      if (resource.linksTo === 'edit') {
        url += '/edit'
      }

      Nova.visit({
        url,
        remote: false,
      })
    },
```


after
```
    goToSelectedResource(event, resource) {
      if (this.canceller !== null) this.canceller()

      this.closeSearch()

      let url = Nova.url(
        `/resources/${resource.resourceName}/${resource.resourceId}`
      )

      if (resource.linksTo === 'edit') {
        url += '/edit'
      }

      if (event.metaKey || event.ctrlKey) {
        window.open(url);
      } else {
        Nova.visit({
          url,
          remote: false,
        })
      }
    },
```


Many Thanks!!

**Reply by crynobone** [reply]
Stop tagging the maintainer on reply.

**Comment by ziming** [comment]
i havent try myself but does middle mouse click work for u?

**Reply by vesper8** [reply]
It does not.. and judging by the code, there's no reason why using any mouse button would have any such effect until my solution is implemented.

**Comment by davidhemphill** [comment, marked as answer]
I've added the ability to hold the `meta` key (same as resource table rows) and click the search result to open them in a new tab. Also, holding `meta` while pressing enter will also open the result in a new tab. 👍

**Reply by davidhemphill** [reply]
This will ship in the next release.

**Reply by vesper8** [reply]
hell yea 💪
